README.packaging

```diff
@@ -6,11 +6,18 @@ Pocl should probably be built with ICD enabled for desktop
 distributions. Pocl does not have an ICD loader, so a dependancy
 on one would be beneficial.
 
-The pocl.icd file (which the ICD loader uses to load the pocl lib)
-has a full path to the installed libpocl.so file. When building
-for a multiarch, remove the "@libdir@/" from pocl.icd.in before
-building. This way the system library loader can pick up the correct
-architecture library.
+CMake options for a distribution build
+--------------------------------------
+- ``-DKERNELLIB_HOST_CPU_VARIANTS=distro``
```

Comment:
I think this is a useful feature to include in the change list.

Reply:
well, it's been in previous releases already, but i see it's not documented. Will fix